elide-integration-tests/src/test/java/com/yahoo/elide/tests/UserTypeIT.java

```diff
@@ -186,25 +181,25 @@ public void testUserTypeMissingUserTypeProperties() throws Exception {
         partialAddress.put("zip", partialZip);
 
         Resource resource = resource(
-                type("person"),
-                id("4"),
-                attributes(
-                        attr("name", "WC"),
-                        attr("address", partialAddress)
-                )
+            type("person"),
```

Comment:
I think we have single tab here but also have double tabs in other places.

Reply:
Double indent seems to intense. Do you agree?

Reply:
I am fine with either way. I just got confused when I saw the inconsistency.